packages/xod-client/src/editor/constants.js

```diff
@@ -95,3 +95,7 @@ export const SIDEBAR_IDS = {
 };
 
 export const PANEL_CONTEXT_MENU_ID = 'PANEL_CONTEXT_MENU_ID';
+
+// Event name for pub/sub to update position of Helpbox
+// while User scrolls ProjectBrowser or selects another Patch
+export const UPDATE_HELPBOX_POSITION = 'UPDATE_HELPBOX_POSITION';
```

Comment:
Should we better call it `SCROLL_PROJECT_BROWSER`? That’s the fact, and updating the helpbox position is one of the reactions (although the only one at the moment)

Reply:
In fact, it listens to scroll and click events, so it's not just "SCROLL"...
But you're right, that's only about updating position from ProjectBrowser.

I propose to leave it as is and rename it to a more appropriate name later (during #932) when it will receive update events from the quick search.